include/zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt_client.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_IMG_MGMT_CLIENT_
+#define H_IMG_MGMT_CLIENT_
+
+#include <inttypes.h>
+#include <zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Image list data.
+ */
+struct mcumgr_image_data {
+	int slot_num;
+	int img_num;
+	char hash[IMG_MGMT_HASH_LEN];
+	char version[IMG_MGMT_VER_MAX_STR_LEN + 1];
+	bool bootable : 1;
+	bool pending : 1;
+	bool confirmed : 1;
+	bool active : 1;
+	bool permanent : 1;
+};
+
+/**
+ * @brief MCUmgr Image list response.
+ *
+ *	Structure image_list_length and image_list pointer are defined by user for store received
+ *	image list data. image_list_length is updated by MCUmgr client for response.
```

Comment:
```suggestion
 *	Structure image_list_length and image_list pointer are defined by user for storing received
 *	image list data. image_list_length is updated by MCUmgr client for response.
```

And the indentation looks off here